Pocket Casts App ClipUITests/Pocket_Casts_App_ClipUITests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+final class Pocket_Casts_App_ClipUITests: XCTestCase {
```

Comment:
Is it worth to keep this default template tests?

Reply:
I was planning to add unit and UI tests later on, although I had quite a bit of trouble getting the invocation URL to work. I'll remove them for now.